src/CookieSessionStore.ts

```diff
@@ -1,21 +1,30 @@
 import { SessionStore } from "./types";
-import JWTSession from "./JWTSession";
+
+export interface CookieSessionStoreOptions {
+    path?: string;
+    sameSite?: string;
```

Comment:
let's define the two valid strings with a union type

Reply:
Changed that and also added the "None" value as as not specifying sameSite will use the browsers default (which will become `SameSite=Lax` in future chrome versions: https://www.chromium.org/updates/same-site)